data/tests/tech/aria/aria-placeholder.json

```diff
@@ -0,0 +1,1693 @@
+              "applied_to": "html/input(type-text)_element",
+              "result": "pass"
+            }
+          ]
+        },
+        {
+          "command": "next_item",
+          "css_target": "#text4",
+          "before": {
+            "mode": "auto",
+            "virtual_location": "before target",
+            "focus_location": "before target"
+          },
+          "after": "target",
+          "output": "\"12 25 dash 2000, content selected, her birthday, edit text\"",
+          "results": [
+            {
+              "feature_id": "aria/aria-placeholder_attribute",
+              "feature_assertion_id": "convey_placeholder_labelled_with_value",
+              "applied_to": "html/input(type-text)_element",
+              "result": "fail"
+            }
+          ]
+        },
+        {
+          "command": "next_focusable_item",
+          "css_target": "#text4",
+          "before": {
+            "mode": "auto",
+            "virtual_location": "before target",
+            "focus_location": "before target"
+          },
+          "after": "target",
+          "output": "\"12 25 dash 2000, content selected, her birthday, edit text\"",
+          "results": [
+            {
+              "feature_id": "aria/aria-placeholder_attribute",
+              "feature_assertion_id": "convey_placeholder_labelled_with_value",
+              "applied_to": "html/input(type-text)_element",
+              "result": "fail"
+            }
+          ]
+        }
+      ]
+    }
+  },
+  "history": [
+    {
+      "date": "2021-11-26",
+      "message": "Test created"
+    }
+  ],
+  "versions": {
+    "jaws": {
+      "browsers": {
+        "chrome": {
+          "at_version": "2021.2105.53",
+          "browser_version": "91",
+          "os_version": "10.0.18363",
+          "date": "2021-11-26"
+        },
+        "firefox": {
+          "at_version": "2021.2105.53",
+          "browser_version": "94",
+          "os_version": "10.0.18363",
+          "date": "2021-11-26"
+        },
+        "edge": {
+          "at_version": "2021.2105.53",
+          "browser_version": "96",
+          "os_version": "10.0.18363",
+          "date": "2021-11-26"
+        }
+      }
+    },
+    "nvda": {
+      "browsers": {
+        "chrome": {
+          "at_version": "2021.2",
+          "browser_version": "91",
+          "os_version": "10.0.18363",
+          "date": "2021-11-26"
+        },
+        "firefox": {
+          "at_version": "2021.2",
+          "browser_version": "94",
+          "os_version": "10.0.18363",
+          "date": "2021-11-26"
+        },
+        "edge": {
+          "at_version": "2021.2",
+          "browser_version": "96",
+          "os_version": "10.0.18363",
+          "date": "2021-11-26"
+        }
+      }
+    },
+    "narrator": {
+      "browsers": {
+        "chrome": {
+          "at_version": "10.0.18363",
+          "browser_version": "91",
```

Comment:
This should be fine, but is it possible test with a newer version?

Reply:
That's what happens when I test on my work machine - very locked down.
I've retested all Windows results on a personal machine instead, much better with the newer versions all round.